Aspire Budgeting/Views/CardView/CardListView.swift

```diff
@@ -5,20 +5,24 @@
 
 import SwiftUI
 
-struct CardListView: View {
+struct DasboardCardsListView: View {
 
-  let cardViewItems: [CardView.CardViewItem]
+  let cardViewItems: [BaseCardView<DashboardCardView>.CardViewItem]
 
   var body: some View {
     GeometryReader {g in
       ScrollView {
         VStack {
           ForEach(0..<self.cardViewItems.count) { idx in
             GeometryReader { geo in
-              CardView(colorInfo: CardListView.colorInfos[idx % CardListView.colorInfos.count],
-                       cardViewItem: self.cardViewItems[idx],
+              BaseCardView<DashboardCardView>(
+//                       cardViewItem: self.cardViewItems[idx],
                        minY: g.frame(in: .global).minY,
-                       curY: geo.frame(in: .global).minY)
+                       curY: geo.frame(in: .global).minY,
+                baseColor: DasboardCardsListView.baseColors[idx % DasboardCardsListView.baseColors.count]) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/5d9f74b3b35e3fb9746b1333ffd07f18323321a5/Aspire Budgeting/Views/CardView/CardListView.swift#L22"></span>Function parameters should be aligned vertically if they’re in multiple lines in a method call.<br />
<code>vertical_parameter_alignment_on_call</code> <code>CardListView.swift:22</code></td>
    </tr>
  </tbody>
</table>